lib/templates/deploy.batch.basic.template

```diff
@@ -3,12 +3,21 @@
 :: Deployment
 :: ----------
 
+setlocal enabledelayedexpansion
```

Comment:
I don't think this is necessary since we were setting it in the prefix file:
https://github.com/projectkudu/KuduScript/blob/725831cf960bbf98884d1c0acb52b353d81f8a39/lib/templates/deploy.batch.prefix.template#L21

Reply:
Yep, that's right. Removed this, pushed an update.